packages/builder/src/stores/portal/auth.js

```diff
@@ -172,6 +172,14 @@ export function createAuthStore() {
         resetCode,
       })
     },
+    generateAPIKey: async () => {
+      const info = await API.generateAPIKey()
+      return info?.apiKey ? info.apiKey : null
```

Comment:
Super minor but we can just shorten this to `return info?.apiKey` (since it's not like it can return literal `false` or anything so a falsey check falling back to `null` won't do anything extra).

Or if this call will only ever return an API key and not anything else, we should move this into `frontend-core` so that the actual API call just returns the string directly.